packages/fxa-auth-client/lib/client.ts

```diff
@@ -1249,6 +1249,24 @@ export default class AuthClient {
     );
   }
 
+  async getRecoveryKeyHint(accountResetToken: hexstring) {
```

Comment:
@vbudhram Should these functions be in `client.ts` or `recoveryKey.ts`? I see recoveryKey functions in both.